src/kata1.fizzBuzz.js

```diff
@@ -1,3 +1,14 @@
-const fizzBuzz = (number) => {};
+const fizzBuzz = (number) => {
```

Comment:
You're almost there with this. Remember, the `%` operator returns the remainder of dividing the first number by the second number. So if the first number is a multiple of the second then the remainder is 0.

24 % 5 === 4
25 % 5 === 0
26 % 5 === 1
27 % 5 === 2
28 % 5 === 3
29 % 5 === 4
30 % 5 === 0
...and so on.

Your `if` statement conditions are performing the remainder operation but are not checking that the result is equal to `0`. I like the way you have used if/return instead of if/else structure. Just a little hint, when it comes to branching logic, the order of checks often matters.